webapp/views/App/views/Analysis/Chain/ChainDEPRECATED.js

```diff
@@ -0,0 +1,81 @@
+import './Chain.scss'
```

Comment:
this is an old file that will be removed afterwards in chain cleanup code

Reply:
hahah nicefile name hahaha